addons/sale_project/tests/test_child_tasks.py

```diff
@@ -221,28 +221,3 @@ def test_linking_on_parent_with_multiple_tasks(self):
         for child in children:
             self.assertEqual(child.sale_line_id, self.order_line)
             self.assertFalse(child.user_ids)
-
-    def test_allow_billable_on_subtasks(self):
-        parent = self.env['project.task'].create({
-            'name': 'Parent Task',
-            'project_id': self.project.id,
-            'child_ids': [
-                Command.create({
-                    'name': 'Subtask 1',
-                    'project_id': self.project.id,
-                }),
-                Command.create({
-                    'name': 'Subtask 2',
-                    'child_ids': [Command.create({'name': 'Subsubtask'})],
-                }),
-            ],
-        })
-        self.assertTrue(all((parent + parent._get_all_subtasks()).mapped('allow_billable')))
-
-        subtask2 = parent.child_ids.filtered(lambda t: t.name == 'Subtask 2')
-        subsubtask = subtask2.child_ids
-        project_non_billable = self.env['project.project'].create({'name': 'Non-billable project', 'allow_billable': False})
-
-        subtask2.project_id = project_non_billable
-        self.assertFalse(subtask2.allow_billable)
-        self.assertFalse(subsubtask.allow_billable)
```

Comment:
What would the result be here ? Shouldn't we keep the test case for that ?

Reply:
Setting a billable project on a task will obviously change its `allow_billable` field as it is a related one.
Setting a billable project on a task's parent will do nothing.

=> Nothing to test, really. :shrug:

Reply:
Sure then :shrug: